docs/advanced/pcidevices.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position: 0
+sidebar_label: PCI Devices
+title: ""
+---
+
+# PCI Devices
+
+A PCIDevice represents a host device with a PCI address. 
+The devices can be passed through the hypervisor to a VM by creating a PCIDeviceClaim, 
+or by using the UI to enable passthrough. Passing a device through the hypervisor means that 
+the VM can directly access the device, and effectively owns the device. A VM can even install 
+it's own drivers for that device.
+
+## Enabling passthrough on a PCI Device
+
+Go to the Advanced -> PCI Devices UI:
+
+![](/img/v1.1/pcidevices/advanced-pcidevices-index.png)
+
+Search for your device by vendor name (e.g. NVIDIA, Intel, etc.) or device name.
+
+![](/img/v1.1/pcidevices/search-pcidevices.png)
+
+Select the devices you want to enable for passthrough:
+
+![](/img/v1.1/pcidevices/select-pcidevices.png)
+
+Then click "Enable Passthrough", and wait until all the devices say "Enabled":
+
+![](/img/v1.1/pcidevices/enable-pcidevices-inprogress.png)
+
+
+![](/img/v1.1/pcidevices/enable-pcidevices-done.png)
+
+## Attaching PCI Devices to a VM
+
+Go to the VM UI and select "Edit Config"
```

Comment:
Does the user need to shut down the VM first?

Reply:
They don't. It will just prompt you to reboot the VM if it's turned on.